### ISSUE TITLE
Issue with Vision AI DevKit Postman Collection

### ISSUE THREAD

**Issue by anilvpr** [microsoft/vision-ai-developer-kit#302]
Followed the steps outlined in Vision AI DevKit Postman Collection Readme. Able to login to the Camera using "Login" method. However the subsequent REST APIs like "Get preview config", "Get preview info (RTSP url) etc. fails despite updating the Environment with sessionId copied from the response for "Login" request.

**Comment by PuneetRahejaMS** [comment]
@anilvpr Do you see any error message

**Comment by anilvpr** [comment]
Summary of my observations:
1. Booted VAIDK. After a while connected the ADB cable and logged into the shell using "adb shell"
2. Checked the IP address "ifconfig wlan0"
3. Log retried using "logcat" command is attached as "logdump_1.txt" 
4. Run vlc player on Host PC and opened the network stream via RTSP
5. Everything is fine so far
6. Now launched Postman App on Host PC
7. Imported "vision-ai-devkit.postman_collection.json"
8. Created new environment and added env variable "hostIp" with value configured to be the IP address of the wlan0 interface of DevKit
9. Loaded and executed "Login" command from Postman
10. Response is 
{
    "status": true
}

command seems to have executed good !!
11. Retrieved the "logcat dump" from DevKit and attached as "logdump_2.txt"
12. Now copied the sessionId from "Set-Cookie" field of Headers of this repsonse
13. Copied this value to the 2nd env variable of the Postman Environment as value for "sessionId"
14. Now loaded the "Get preview config" command in Postman and hit on "Send" button
15. Response in Postman is as follows:
{
    "resolutionSelectVal": 1,
    "resolution": [
        "4K",
        "1080P",
        "720P",
        "480P"
    ],
    "encodeModeSelectVal": 1,
    "encodeMode": [
        "HEVC/H.265",
        "AVC/H.264"
    ],
    "bitRateSelectVal": 3,
    "bitRate": [
        "512Kbps",
        "768Kbps",
        "1Mbps",
        "1.5Mbps",
        "2Mbps",
        "3Mbps",
        "4Mbps",
        "6Mbps",
        "8Mbps",
        "10Mbps",
        "20Mbps"
    ],
    "fpsSelectVal": 1,
    "fps": [
        24,
        30
    ],
    "displayOut": 1,
    "status": true
}

16. Now loaded the "Get preview info (RTSP url)" command in Postman and hit on "Send" button
17. Response in Postman is
{
    "status": false
}

18. "logcat" command does not show output on shell
19. VLC Player stops streaming

**Comment by PuneetRahejaMS** [comment]
@anilvpr Thanks for sharing the details. Allow me some time to get back

**Comment by jkubicka** [comment]
Hi @drwill-ms There seems to be an issue with Postman Collection you committed awhile ago - any idea on how to resolve?

**Comment by drwill-ms** [comment]
PostMan is the method for sending the request to the camera. It does not control the camera's response.

The only hitch I ran into when using the PostMan methods, was that the camera sometimes expires sessionId. When that happens, one must log in again, and update their sessionId.

**Comment by jkubicka** [comment]
Thanks for the response @drwill-ms Any idea why REST APIs like "Get preview config", "Get preview info (RTSP url) etc. fails would fail?

@anilvpr Are you still seeing this issue with these REST APIs?

**Comment by drwill-ms** [comment]
I loaded my camera up and tried the repro steps. I also got a false status when running "Get preview info". I tried "Set preview on/off", which came back status true, and then reran "Get preview info" which succeeded. The postman collection seems to work, but the REST API seems to be finicky in the order in which calls can be made.

I don't know why the camera behaves that way. We don't own that REST API. :(
